book/src/zcash/ywallet-demo.md

```diff
@@ -11,6 +11,14 @@ the transaction plan with a command line tool, using FROST.
 
 This tutorial assumes familiarity with the command line.
 
+This tutorial uses:
+1. RedPallas
+2. YWallet
+3. Sprout
+4. [Sapling](https://docs.rs/reddsa/0.5.1/reddsa/sapling/index.html)
+5. Trusted dealer journey
+6. [frost-ed25519 crate](https://crates.io/crates/frost-ed25519)
+
```

Comment:
This seems a bit confusing for the user, it feels like they would expect to do something about this list but it just provides context. I think we could skip this, or do you think this information is essential for the tutorial?

Reply:
I'll see if I can rewrite this or I'll just remove it